app/views/step_mailer/proposal_notification.text.erb

```diff
@@ -2,6 +2,8 @@
     requester_name: @proposal.requester.full_name,
     step_type_noun: @step.noun) %>
 
+<%= t("mailer.step_mailer.proposal_notification.step_status_html") %>
```

Comment:
if this locale has `html` in it, we want to use something else for the text-only version of the email